source/whats-new.txt

```diff
@@ -55,6 +55,19 @@ What's New in 2.0
    :ref:`Connection Example Code <go-connection-example-code>` in the
    Connection Guide.
 
+The 2.0 {+driver-short+} release includes the following improvements and fixes:
+
+- Updates to monitoring event documents:
+
+  - The ``CommandStartedEvent`` and ``CommandFinishedEvent`` structs
+    have a single ``ServerConnectionID`` field of type ``int64`` to
+    capture the connection ID.
+    
+  - Changes the type of the ``ConnectionID`` field in ``PoolEvent``
+    structs to ``int64`` from ``uint64``.
```

Comment:
I: [Start all items with fragments or all items with sentences](https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items)

```suggestion
  - The ``ConnectionID`` field in ``PoolEvent``
    structs now use the ``int64`` data type instead of ``uint64``.
```